pkg/api/metadata.go

```diff
@@ -69,14 +85,21 @@ func (m *Metadata) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 // ServeHTTP sends API version v1 documentation response.
 func (v *VersionMetadata) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	version, err := getCommitSha()
+	if err != nil {
+		shared.HandleError(r, w, err.(*errors.ServiceError))
+		return
+	}
+
 	// Set the content type:
 	w.Header().Set("Content-Type", "application/json")
 
 	// Prepare the body:
 	body := VersionMetadata{
-		ID:   v.ID,
-		Kind: "APIVersion",
-		HREF: r.URL.Path,
+		ID:            v.ID,
+		Kind:          "APIVersion",
+		HREF:          r.URL.Path,
+		ServerVersion: version,
```

Comment:
According to the `getCommitSha` implementation, `version` here could be an empty string. Is that correct?

Reply:
@machi1990 WDYT?

Reply:
That's a good question.
It should never happen in our case :tm: but it is theoretically possible to not receive a commit sha. If the binary is built from outside the directory containing the source code then the commit information would not be there. I don't know however in that case if `ReadBuildInfo` would return an error or not. I suspect it wouldn't return an error and simply the `vcs.revision` key would not exist or be empty

Reply:
+1